### DISCUSSION TITLE
If it is possible to set width for one specific page (eg. index.md)

### DISCUSSION THREAD

**Discussion by xihajun** [squidfunk/mkdocs-material#4797] [Q&A] [answered]
I am thinking to have a specific page without having the right sidebar (table of contents) which will free up more space. It might be able to be done by set a larger content width. I can find how to set a Content area width.
<img width="722" alt="image" src="https://user-images.githubusercontent.com/25631641/209350498-5e78cd92-6b13-4654-99a8-6cf29c76ffc7.png">

However, I wonder if it is possible to set it for only one page instead of all the pages?

Many thanks

**Comment by squidfunk** [comment, marked as answer]
Yes, just add it to the Markdown document itself:

``` markdown
<style>
  .md-grid {
    max-width: 1440px;
  }
</style>

# Page headline

...
```

**Reply by xihajun** [reply]
Thank you so much!
